modules/Db2/scripts/install_Db2.sh

```diff
@@ -14,11 +14,18 @@
 CUR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 OC_CMD=oc
 
-echo
+. ../../modules/Db2/scripts/common-ocp-utils.sh
```

Comment:
This should be `./common-ocp-utils.sh` since it resides in the same directory as this file.  Unnecessary to go up so far in the tree only to come back down again.